crates/rustc_codegen_c_ast/src/pretty.rs

```diff
@@ -0,0 +1,131 @@
+//! Pretty printing
+
+// TODO: pretty printing test
```

Comment:
In the interest of time it is okay to not do the most fine-grained testing, but something is needed here. I would suggest you just add some simple output tests; in each module you can have a couple small tests that just construct an AST and print it to a string, then compare against a saved file. If `RUSTC_CG_C_BLESS` is set, overwrite the file.